README.rst

```diff
@@ -42,13 +42,35 @@ Building
 There are different ways to compile python-blosc, depending if you want
 to link with an already installed Blosc library or not.
 
-Compiling with an installed Blosc library (recommended)
--------------------------------------------------------
 
-Python and Blosc-powered extensions have a difficult relationship when
-compiled using GCC, so this is why using an external C-Blosc library is
-recommended for maximum performance (for details, see
-https://github.com/Blosc/python-blosc/issues/110).
+Installing via setuptools
+-------------------------
+
+`setuptools` is limited to using the compiler specified in the environment 
+variable `CC` which on posix systems is usually `gcc`. This often causes 
+trouble with the Snappy codec, which is written in C++, and as a result Snappy
+is no longer compiled by default. This problem is not known to affect MSVC or 
+clang. Snappy is considered optional in Blosc as its compression performance 
+is below that of the other codecs.
+
+Any codec can be enabled (`=1`) or disabled (`=0`) on this build-path with the appropriate
+OS environment variables `INCLUDE_LZ4`, `INCLUDE_SNAPPY`, `INCLUDE_ZLIB`, and 
+`INCLUDE_ZLIB`. Snappy is disabled by default on posix systems.
```

Comment:
Reading your PR it seems to me that Snappy is disable by default for *all* the platforms.  I'd put:

```
By default all the codecs in Blosc are enabled except Snappy (due to some issues with C++ in posix systems).
```

Reply:
Ping?  Is that correct or am I misinterpreting this?